Kernel/Arch/aarch64/Processor.cpp

```diff
@@ -83,8 +84,29 @@ u32 Processor::smp_wake_n_idle_processors(u32 wake_count)
 
 void Processor::initialize_context_switching(Thread& initial_thread)
 {
-    (void)initial_thread;
-    TODO_AARCH64();
+    VERIFY(initial_thread.process().is_kernel_process());
+
+    m_scheduler_initialized = true;
+
+    // FIXME: Figure out if we need to call {pre_,post_,}init_finished once aarch64 supports SMP
+    Processor::set_current_in_scheduler(true);
+
+    auto& regs = initial_thread.regs();
+    // clang-format off
+    asm volatile(
+        "mov sp, %[new_sp] \n"
```

Comment:
(what's with the spaces before the `\n`s?)

Reply:
Copied this style from the existing `x86_64` code :shrug: I guess this made it a bit easier to read. I can remove it if you really want that :)